specification/azurefleet/AzureFleet.Management/baseVirtualMachineProfile.tsp

```diff
@@ -0,0 +1,2518 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Microsoft.AzureFleet;
+
+/**
+ * Describes the base virtual machine profile for fleet
+ */
```

Comment:
user facing docs need to be `@doc` and not just doc comments, right?

Reply:
Is it documented any where? The [tool](https://azure.github.io/typespec-azure/docs/howtos/migrate-swagger/get-started) which converts swagger to typespec, generated the doc comments in this way.

Reply:
I made the changes in typespec to use `doc` instead of comments but there is no change in swagger. Hence keeping it the same. 
![image](https://github.com/Azure/azure-rest-api-specs/assets/7030203/45a6a3dc-e218-458a-b544-6d37edaf0c9f)
![image](https://github.com/Azure/azure-rest-api-specs/assets/7030203/c493e5cf-3d96-432b-9b15-b3c87f84be95)